src/Helper.php

```diff
@@ -469,4 +474,51 @@ public function getSpreadsheetWorksheets() : object
         return (object)$sheets;
 	} 
 
-}
+     /**
+     * copy sheet in same spreadsSheet
+     * @return void
+     * 
+     */
+    public function duplicateSheetInSameSpreadsheet(?string $nameOrigin = null, ?string $nameDestiny) : void
```

Comment:
- Cambia el nombre de la funcion a `duplicateSheet`
- que solo le puedas pasar el nombre del nuevo sheet creado. El $nameOrigin debe ser `$this->getWorksheetName()`

Reply:
Despues si puedes crear una funcion estatica que se llame  `duplicateSheetInSameSpreadsheet` que use la funcion nueva para hacer esto que haces ahorita de copiar un worksheet segun el nombre que le pases.